tests/SAML2/XML/mdui/UIInfoTest.php

```diff
@@ -215,15 +231,15 @@ public function testUnmarshalling(): void
         );
 
         $this->assertCount(2, $uiinfo->getDisplayName());
-        $this->assertEquals('University of Examples', $uiinfo->getDisplayName()['en']);
-        $this->assertEquals('Univërsitä øf Exåmpleß', $uiinfo->getDisplayName()['el']);
+        $this->assertEquals('University of Examples', $uiinfo->getDisplayName()[0]->getValue());
+        $this->assertEquals('Univërsitä øf Exåmpleß', $uiinfo->getDisplayName()[1]->getValue());
         $this->assertCount(2, $uiinfo->getInformationURL());
-        $this->assertEquals('http://www.example.edu/en/', $uiinfo->getInformationURL()['en']);
-        $this->assertEquals('http://www.example.edu/', $uiinfo->getInformationURL()['el']);
+        $this->assertEquals('http://www.example.edu/en/', $uiinfo->getInformationURL()[0]->getValue());
+        $this->assertEquals('http://www.example.edu/', $uiinfo->getInformationURL()[1]->getValue());
         $this->assertCount(1, $uiinfo->getPrivacyStatementURL());
-        $this->assertEquals('https://example.org/privacy', $uiinfo->getPrivacyStatementURL()['en']);
+        $this->assertEquals('https://example.org/privacy', $uiinfo->getPrivacyStatementURL()[0]->getValue());
         $this->assertCount(1, $uiinfo->getDescription());
-        $this->assertEquals('Just an example', $uiinfo->getDescription()['en']);
+        $this->assertEquals('Just an example', $uiinfo->getDescription()[0]->getValue());
```

Comment:
Should we also check the language tag for each object here?

Reply:
Probably, yes!  I'l take care of it